lexforge/build.gradle.kts

```diff
@@ -17,5 +17,5 @@ unimined.minecraft {
 dependencies {
 	compileOnly(project(":stubs"))
 
-	"minecraftLibraries"("blue.endless:jankson:${"jankson_version"()}")
+	"minecraftLibraries"("dev.nolij:zson:${"zson_version"()}")
```

Comment:
This can probably be moved to the main `build.gradle.kts` with an `implName in lexForgeImpls` check or something like that.